certified-connectors/Cloud BOT/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "<client-id>",
+          "scopes": [
+            "execute",
+            "refer",
+            "suspend"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False"
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://console.c-bot.pro/oauth/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://console.c-bot.pro/oauth/token"
+            },
+            "refreshUrl": {
+              "value": "https://console.c-bot.pro/oauth/token"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ffffff",
```

Comment:
Unfortunately, pure white (#ffffff) isn't an allowed color here. It can be an off-white if you would like, but not #ffffff.

Reply:
Fixed the color of the icon.